sbc_compassion/models/sponsorship_correspondence.py

```diff
@@ -259,6 +265,37 @@ def _set_partner_review(self):
             if letter.correspondant_id.mandatory_review:
                 letter.mandatory_review = True
 
+    @api.depends('page_ids')
+    def _compute_page(self):
+        self.original_text = self._get_original_text('original_text')
+        self.translated_text = self._get_original_text('translated_text')
+        self.nbr_pages = len(self.page_ids)
+
+    def _inverse_page(self):
+        pdb.set_trace()
+        if len(self.page_ids) > 0:
+            # Keep only the first page and remove the other
+            self.page_ids[0].write({
+                'original_text': self.original_text,
+                'translated_text': self._get_original_text('translated_text')})
+            self.page_ids[1:].unlink()
+        else:
+            self.page_ids.create(
+                {'sponsorship_correspondence_id': self.id,
+                 'original_text': self.original_text,
+                 'translated_text': self.translated_text})
+
+    def _get_original_text(self, source_text):
+        if len(self.page_ids) > 1:
```

Comment:
You don't need to keep your tests with len(pages), as the line I gave you should already handle each cases.